### PR TITLE
Update clipboard and add copy namespace functionality

### PR DESCRIPTION
**Pull request by minghorn**
* Updated `CopyToClipboard` to use `navigator.clipboard` instead of the deprecated `execCommand`
* Clicking the object key or array key name will copy the path to clipboard
* Upgraded `enzyme` to fix test failures.

### CODE REVIEW

File: src/js/helpers/util.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import regeneratorRuntime from 'regenerator-runtime';
```
Comment (clangen-nw) [inline src/js/helpers/util.js:1]: Hmm, why is this necessary?

**Review by clangen-nw — APPROVED** [review]
LGTM -- just one question about an import that looks like it may be auto-generated and unnecessary.

File: src/js/helpers/util.js (hunk 1 above)
Comment (minghorn) [reply src/js/helpers/util.js:1]: The tests fail if I don't import regenerator-runtime, I'm not totally sure why though. The failures say that regeneratorRuntime is undefined, then when I import it here the error goes away 🤷‍♀️
